pontoon/batch/actions.py

```diff
@@ -125,13 +140,14 @@ def replace_translations(form, user, translations, locale):
     latest_translation_pk = None
 
     try:
-        old_translations, changed_translations = utils.find_and_replace(
+        old_translations, changed_translations, invalid_translation_pks = utils.find_and_replace(
             translations,
             find,
             replace,
             user
         )
         changed_translation_pks = [c.pk for c in changed_translations]
+
         if changed_translation_pks:
             latest_translation_pk = max(changed_translation_pks)
     except Translation.NotAllowed:
```

Comment:
You can remove this try - except block now that we no longer raise `Translation.NotAllowed`. In fact, you can even remove the `TranslationNotAllowed` exception from `pontoon.base.models`.